### ISSUE TITLE
Docker build error

### ISSUE THREAD

**Issue by ghost** [felixbiessmann/fipi#1]
I build, then compose but get the following..

<img width="1152" alt="screen shot 2016-07-07 at 12 38 05 am" src="https://cloud.githubusercontent.com/assets/18179725/16621864/1e2caf36-43db-11e6-975c-cd7d490a496c.png">

**Comment by felixbiessmann** [comment]
yes, i ran into this as well. i tried for some time to fix it but didn't succeed yet in the limited amount of time i had so far. one documented solution appears to be to install libgfortran using conda separately, but this hasn't worked for me. as soon as i'll have some time for this, i'll try and refactor the entire docker build to use less of a mix of pip/conda.

**Comment by ghost** [comment]
Seems like its a common issue. https://github.com/travis-ci/travis-ci/issues/4816

**Comment by felixbiessmann** [comment]
exactly, that's what i was referring to https://github.com/travis-ci/travis-ci/issues/4816#issuecomment-148789525
